smf/clickhouse_server/method_script.sh

```diff
@@ -10,4 +10,4 @@ set -o pipefail
 # Otherwise, it does not automatically detect the configuration files, nor does
 # it append them when necessary
 cd /opt/oxide/clickhouse_server/
```

Comment:
Same question about this `cd`

Reply:
Hm, looks like it's not. I had added the `cd` because the template in `config.d/` is not "complete" it has overrides on the default ClickHouse template. My understanding from reading the clickhouse docs is that if you use the `--config` flag then you don't merge your template with the default one and only the configuration settings from your file are used. So, if we want to proceed this way, we probably need to make sure we've copied over all default settings to this template. That said, I could be wrong, the docs are a little obscure.

Reply:
We only need configuration that we generate. If we need to be specific about certain values we can add those. I anticipate we'll add a change to keeper retries used for reconfiguration due to testing, but that's probably it for a while.

Reply:
Removed in [3a6b659](https://github.com/oxidecomputer/omicron/pull/6894/commits/3a6b6592f1a8a02816ab1ac7495b98e56164b9b4)